app/models/product.rb

```diff
@@ -43,6 +43,11 @@ def update_multimedia(multimedia_params)
     end
   end
 
+  def main_display_image_s3_presigned_url(style = nil)
+    object_presigned_url(main_display_image, style)
+  end
+
```

Comment:
I believe we could drop this into the `Innovation` model file and remove it from `Practice`